.github/workflows/linting.yml

```diff
@@ -162,7 +162,7 @@ jobs:
       - name: Run check
         uses: codespell-project/actions-codespell@406322ec52dd7b488e48c1c4b82e2a8b3a1bf630  # master
         with:
-          skip: ./src/control/vendor,./src/control/go.sum,./.git
+          skip: ./src/control/vendor,./src/control/go.sum,./.git,./utils/*.patch
```

Comment:
I add these here to reduce potential conflicts between google/2.6 and master branch